test/rebar_covertool_tests.erl

```diff
@@ -74,7 +72,7 @@ expected_cover_generated_files() ->
 
 -define(rebar_config,
         ["{covertool_app_name, 'otherapp'}.\n",
-         "{lib_dirs, [\"../../../\"]}.\n",
+         "{lib_dirs, [\"../lib\"]}.\n",
```

Comment:
Big plus with this change: now *only* covertool itself gets included here, not covertool and all other OTP apps that are peers of your git checkout of covertool. Nice.